Sage/carbon
# feat(switch): add i18n support

### Proposed behaviour
Add i18n translation support to switch using the `switch.on` and `switch.off` translation keys.

### Current behaviour
The switch component only displays the English ON/OFF.

### Checklist


<del>- [ ] Cypress automation tests
<del>- [ ] Theme support
<del>- [ ] Typescript `d.ts` file added or updated

No need for Cypress, fully unit tested.

### Additional context
Fixes #2722, FE-2628

### Testing instructions

- npm start
- See the default translation here http://localhost:9001/?path=/story/experimental-switch--default
- See the example translation here http://localhost:9001/?path=/story/experimental-switch--default&locale=fr
